.github/workflows/check_vpn.yml

```diff
@@ -0,0 +1,29 @@
+name: Check connect vpn
```

Comment:
Not needed here, we can test with the actual workflow.  Please remove before merging.

Reply:
Are you suggesting to not include the Check-VPN workflow file now and leave it as part of the integration phase?